src/services/syk-inn-api/SykInnApiService.ts

```diff
@@ -52,4 +52,15 @@ export const sykInnApiService = {
             },
             responseSchema: ExistingSykmeldingSchema,
         }),
+    getTidligereSykmeldinger: async (ident: string): Promise<ExistingSykmelding[] | ApiFetchErrors> =>
+        fetchInternalAPI({
+            api: 'syk-inn-api',
+            path: `/api/v1/sykmelding`,
+            method: 'GET',
+            headers: {
+                'Content-Type': 'application/json',
+                'X-IDENT': ident,
+            },
+            responseSchema: ExistingSykmeldingSchema.array(),
+        }),
```

Comment:
API-et her burde egentlig ikke tillate å hente sykmeldinger kun på fnr/dnr. Det burde være kombinasjon av fnr+hpr. Det er jo også en syk-inn-api endring.

Reply:
Tenker du da at, vi bare skal visse sykmeldinger som er utskrevet av hpr?

Reply:
Kvifor tenker du at en skal slå opp på HPR ?  eller er det for auditlogg?

Reply:
eller njæ kanskje jeg tenker feil, det er jo forsåvidt nice å se alle sykmeldinger uavhengig av HPR, i hvert fall nå i "screw legal" utforskingssteget.